locale/en/download/package-manager.md

```diff
@@ -59,20 +59,20 @@ pacman -S nodejs npm
 Node.js is available as a module called `nodejs` in CentOS/RHEL 8 and Fedora.
 
 ```bash
-dnf module install nodejs:<stream>
+dnf module install nodejs:<stream>/<profile>
 ```
 
 where `<stream>` corresponds to the major version of Node.js.
-To see a list of available streams:
+To see a list of available streams and profiles:
 
 ```bash
 dnf module list nodejs
 ```
 
-For example, to install Node.js 12:
+For example, to install Node.js 16:
 
 ```bash
-dnf module install nodejs:12
+dnf module install nodejs:16/default
```

Comment:
This example looks like it will only work on Fedora. 

i.e. Fedora 35 has a profile called `default`:
```console
# dnf module list nodejs
Last metadata expiration check: 0:14:53 ago on Tue Dec  7 15:48:39 2021.
Fedora Modular 35 - x86_64
Name                          Stream                        Profiles                                                Summary
nodejs                        12                            default [d], development, minimal                       Javascript runtime
nodejs                        14                            default, development, minimal                           Javascript runtime
nodejs                        15                            default, development, minimal                           Javascript runtime
nodejs                        16                            default, development, minimal                           Javascript runtime

Fedora Modular 35 - x86_64 - Updates
Name                          Stream                        Profiles                                                Summary
nodejs                        12                            default [d], development, minimal                       Javascript runtime
nodejs                        14                            default, development, minimal                           Javascript runtime
nodejs                        15                            default, development, minimal                           Javascript runtime
nodejs                        16                            default, development, minimal                           Javascript runtime

Fedora Modular 35 - x86_64 - Test Updates
Name                          Stream                        Profiles                                                Summary
nodejs                        12                            default [d], development, minimal                       Javascript runtime
nodejs                        14                            default, development, minimal                           Javascript runtime
nodejs                        15                            default, development, minimal                           Javascript runtime
nodejs                        16                            default, development, minimal                           Javascript runtime

Hint: [d]efault, [e]nabled, [x]disabled, [i]nstalled
#
```

but CentOS 8 does not:
```console
# dnf module list nodejs
Failed to set locale, defaulting to C.UTF-8
Last metadata expiration check: 0:03:37 ago on Tue Dec  7 16:01:20 2021.
CentOS Linux 8 - AppStream
Name                         Stream                       Profiles                                                   Summary
nodejs                       10 [d]                       common [d], development, minimal, s2i                      Javascript runtime
nodejs                       12                           common [d], development, minimal, s2i                      Javascript runtime
nodejs                       14                           common [d], development, minimal, s2i                      Javascript runtime
nodejs                       16                           common [d], development, minimal, s2i                      Javascript runtime

Hint: [d]efault, [e]nabled, [x]disabled, [i]nstalled
#
```

and neither does UBI 8:
```console
# dnf module list nodejs
Updating Subscription Management repositories.
Unable to read consumer identity

This system is not registered to Red Hat Subscription Management. You can use subscription-manager to register.

Last metadata expiration check: 0:00:02 ago on Tue Dec  7 16:05:41 2021.
Red Hat Universal Base Image 8 (RPMs) - AppStream
Name                         Stream                       Profiles                                                   Summary
nodejs                       10 [d]                       common [d], development, minimal, s2i                      Javascript runtime
nodejs                       12                           common [d], development, minimal, s2i                      Javascript runtime
nodejs                       14                           common [d], development, minimal, s2i                      Javascript runtime
nodejs                       16                           common [d], development, minimal, s2i                      Javascript runtime

Hint: [d]efault, [e]nabled, [x]disabled, [i]nstalled
#
```

However both CentOS 8 and UBI 8 list the `common` profile as default `[d]` (i.e. if no profile is specified this one will be used) -- I'm not entirely sure why Fedora only has a default `[d]` profile for Node.js 12 and no default for the later versions.

cc @kasicka

Reply:
Default `[d]` profiles are specified in https://pagure.io/releng/fedora-module-defaults, but seems like nobody updated those in a while. The reason for different naming is that there was a proposal to globally rename `default` profiles to `common`, which wasn't accepted, but was already implemented RHEL/CentOS. We'll rename them to common in Fedora too.

Reply:
Fedora is being updated so that it uses common and that the LTS version are marked with [d].

14, and 16 are already fixed in the latest Fedora containers.